### ISSUE TITLE
[Global] "Your corpse is not in that instance"

### ISSUE THREAD

**Issue by Vishiswaz** [Stormforge-gg/Bugtracker-Fordring#1541]
[//]: # (Enclose links to things related to the bug using http://wowhead.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([Quest][Zone] A Vision of the Past)
[//]: # ([NPC] Thoralius the Wise)
[//]: # ([Spell][Class] Frostfire Bolt)
[//]: # ([Talent][Class] Brain Freeze)
[//]: # ([Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([Npc][Drop] Thoralius the Wise)
[//]: # ([Web] Armory doesnt work)

**Description**: 
I died in an RDF group (Slave Pens) and I was kicked from the group before releasing spirit. Instead of just being put back, alive, where I was teleported in, or being able to res by walking into the instance (I got the error "your corpse is not in that instance", I instead was being told by the game to walk back to my corpse, which was located high in the air where I had been on my flying mount before teleporting into the instance. This forced me to spirit res.

Completely screwed up. No wonder people have been complaining about not being able to res.

I believe the correct behavior is one of these options:

1. Walk back to the instance as a corpse, then get immediately ported to wherever you were when you ported into rdf, alive, on a mount if you had been on one.
2. Removal from the group resurrects you automatically and then places you back at your teleport-in point.

**Comment by Weuyn** [comment]
pretty sure its supposed to just ress you at the dungeon entrance in this case.

**Comment by Vishiswaz** [comment]
> pretty sure its supposed to just ress you at the dungeon entrance in this case.

If that's the blizzlike behavior, then that is also stupid. When you are no longer in a dungeon group, you should return to your teleport in location. If you want people to first do the death walk back to the instance, fine. But then the result afterwards should be the same.

**Comment by fondlez** [comment]
Near launch, after the very first pull, I was kicked from a Mana Tombs RFD group exactly as I released - so it was not clear whether it was just before or just after I clicked Release Spirit. Spirit walking back to the entrance ressed me at the entrance, I believe outside the entrance (though it could have been inside the entrance of that Normal instance and then I just walked out). Now I was in the world zone for the instance (Terokkar Forest).

An argument could be made that this effectively allows players to use RFD for teleporting around the world, which probably is not the intent. So, Vishiswaz is almost certainly correct in that the behavior that results in removal from RFD should preserve your original teleport location and presumably always alive.
